e2e/node/acp.test.ts

```diff
@@ -124,22 +149,123 @@ describe("An ACP Solid server", () => {
       { read: true },
       fetchOptions
     );
-    expect(agentAccess).toStrictEqual({
+    expect(agentAccess).toStrictEqual(READ_ONLY_ACCESS);
+    expect(
+      await getAgentAccess(sessionResource, agent, fetchOptions)
+    ).toStrictEqual(READ_ONLY_ACCESS);
+  });
+
+  it("can get and set access for multiple agents", async () => {
+    const alice = "https://example.org/alice";
+    const bob = "https://example.org/bob";
+    const aliceAccess = await setAgentAccess(
+      sessionResource,
+      alice,
+      { read: true },
+      fetchOptions
+    );
+    const bobAccess = await setAgentAccess(
+      sessionResource,
+      bob,
+      { append: true },
+      fetchOptions
+    );
+    expect(bobAccess).toStrictEqual(APPEND_ONLY_ACCESS);
+    expect(aliceAccess).toStrictEqual(READ_ONLY_ACCESS);
+    expect(
+      await getAgentAccess(sessionResource, alice, fetchOptions)
+    ).toStrictEqual(READ_ONLY_ACCESS);
+    expect(
+      await getAgentAccess(sessionResource, bob, fetchOptions)
+    ).toStrictEqual(APPEND_ONLY_ACCESS);
+
+    const bobUpdatedAccess = await setAgentAccess(
+      sessionResource,
+      bob,
+      { read: true },
+      fetchOptions
+    );
+    expect(bobUpdatedAccess).toStrictEqual({
       read: true,
-      append: false,
+      append: true,
       write: false,
       controlRead: false,
       controlWrite: false,
     });
     expect(
-      await getAgentAccess(sessionResource, agent, fetchOptions)
+      await getAgentAccess(sessionResource, bob, fetchOptions)
     ).toStrictEqual({
       read: true,
+      append: true,
```

Comment:
Should we just define a READ_AND_APPEND_ACCESS object at the top too since thats what this is being validated for.